clang/include/clang/Sema/Sema.h

```diff
@@ -891,6 +898,12 @@ class Sema final : public SemaBase {
                                        SourceLocation NewLoc,
                                        SourceLocation OldLoc);
 
+  /// Potentially add a FunctionDecl or BlockDecl to DeclsWithEffectsToVerify.
+  void maybeAddDeclWithEffects(const Decl *D, const FunctionEffectsRef &FX);
```

Comment:
This function is placed is `Sema.cpp` section `Sema.h`, but it's implemented in `SemaDecl.cpp`. You should make sure that other functions you add to `Sema.h` follow this guideline, too.

Reply:
Thanks, did this, including moving some of the other recent, related additions.